colink/instant_server.py

```diff
@@ -88,6 +98,27 @@ def get_colink(self):
         return CoLink(coreaddr=f"http://127.0.0.1:{self.port}", jwt=self.host_token)
 
 
+class InstantRegistry:
```

Comment:
since we are using Python here, can we just do
```class InstantRegistry(InstantServer)```
then just overwrite the new() function?

Reply:
Sure, I will inherit InstantServer class and overwrite new() and clean().